admin/dbEdit.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+require ('dbStream.php');
+
+if (array_key_exists ('name', $_POST)) {
```

Comment:
what about all the other keys? like link etc

Reply:
This is just to check which form has been submitted, rather than to check all the contents of the form.